stl/inc/xlocnum

```diff
@@ -699,10 +918,10 @@ private:
 
         const auto _Dlen = static_cast<size_t>(_Base == 0 || _Base == 10 ? 10 : _Base == 8 ? 8 : 16 + 6);
         string _Groups(1, static_cast<char>(_Seendigit));
-        size_t _Group = 0;
+        size_t _Group_index = 0;

```

Comment:
I know this is pre-existing, but the fact that `_Group_index` is `_Groups.size() - i`, instead of `i` directly is _extremely_ confusing, and I would appreciate _at least_ a comment, and would prefer `_Group_index` to be `i` directly.

Reply:
I'm currently using `_Groups_arr_idx` as the name, although I don't think it's good enough. I think the name shouldn't be `_Idx` because `_Idx` is already used.